### PR TITLE
Fixed: Add a BagIt example

### LINKED ISSUE #189: BagIt example
once @zimeon updates JSON, create an example object using BagIt

### PR DESCRIPTION
**Pull request by ahankinson**
This PR adds a BagIt example.

Fixes #189

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -953,6 +965,127 @@ <h2>Versioned OCFL Object</h2>
...
+        └── myfirstbag
+            ├── data
+            │   └── 27614-h
+            │       └── images
+            │           ├── q173.png
+            │           └── q173.txt
+            └── manifest-md5.txt
+</pre>
+        <p>
+            The state of the object at version 2 would resemble the following BagIt object:
```
Comment (zimeon) [inline draft/spec/index.html]: "resemble" sounds to weak to me. the state really **is** that Bagit object. Perhaps "be"  or "match"?

File: draft/spec/index.html [hunk 2]
```diff
@@ -953,6 +965,127 @@ <h2>Versioned OCFL Object</h2>
 
     <section id="example-bagit-in-ocfl">
         <h2>BagIt in an OCFL Object</h2>
+        <p>
+            [[!BagIt]] is a popular file packaging specification, but unlike OCFL it does not provide a mechanism for
```
Comment (awoods) [inline draft/spec/index.html]: I believe this should be `[[BagIt]]` instead of the normative `[[!BagIt]]`.

Comment (awoods) [inline draft/spec/index.html]: I would advocate for less subjective language such as "popular". Maybe replace "popular" with "common"?

Comment (ahankinson) [reply draft/spec/index.html]: Oh, I thought it was the other way around. I'll fix it.

Comment (ahankinson) [reply draft/spec/index.html]: Awesome? Tubular? Totally Rad? 

OK.

**Pull request merged**